### PR TITLE
ViewBox, Slider, Separator, ScrollViewer, RepeatButton, RadioButton, ProgressBar, ButtonSpinner

### PR DESCRIPTION
**Pull request by uxsoft**

### CODE REVIEW

File: src/Avalonia.FuncUI.DSL/ScrollViewer.fs [hunk 1]
```diff
@@ -17,15 +17,15 @@ module ScrollViewer  =
     type ScrollViewer  with            
 
         /// <summary>
-        /// Gets or sets the vertical scrollbar visibility.
+        /// Sets the vertical scrollbar visibility.
         /// </summary>
-        static member verticalScrollBarVisibility<'t when 't :> ScrollViewer>(value: ScrollBarVisibility) : IAttr<'t> =
+        static member verticalScrollBarVisibility<'t when 't :> Control>(value: ScrollBarVisibility) : IAttr<'t> =
```
Comment (FoggyFinder) [inline src/Avalonia.FuncUI.DSL/ScrollViewer.fs]: why?

Comment (JaggerJo) [reply src/Avalonia.FuncUI.DSL/ScrollViewer.fs]: Ohh, missed that. I guess it's a attached property and should be an extension ob control

Comment (uxsoft) [reply src/Avalonia.FuncUI.DSL/ScrollViewer.fs]: Yep, it's attached. Sometimes you don't have the scrollviewer in your code but you still want to hide it.

Comment (JaggerJo) [reply src/Avalonia.FuncUI.DSL/ScrollViewer.fs]: I'll change that on master with my next PR.

**Pull request merged**